docs/Installation.md

```diff
@@ -227,15 +227,39 @@ Make sure Zonemaster-GUI is properly installed.
 3. Verify that when you mouse over this text the versions of the following
    Zonemaster components are shown: Backend, Engine and GUI.
 
-
-
 ## What to do next?
 
  * For a JSON-RPC API, see the Zonemaster::Backend [JSON-RPC API] documentation.
  * For a command line interface, follow the [Zonemaster::CLI installation] instruction.
  * For a Perl API, see the [Zonemaster::Engine API] documentation.
  * For Https, see [Let's Encrypt / Certbot].
 
+## Serving the GUI and API from a custom base url
+
+In some cases you may want to customize the application to change the base URL
+from wich the GUI is served. To achieve that you will need to change the
+`<base>` in the `index.html`.
+
+Locate the line `<base href="/">` and replace the `href` value with the path you
+want to server the GUI from, e.g. `<base href="/zonemaster/">`. Don't forget the
+trailing `/`.
+
+When serving the application from a different base, you will also need to change
+the Web server configuration by adding an `Alias` directive after the proxy ones.
+Combined with the change of the [API endpoint location] this gives something
+like:
+
+```apache
+ProxyPass /zonemaster/api http://localhost:5000/
+ProxyPassReverse /zonemaster/api http://localhost:5000/
+ProxyPreserveHost On
+
+Alias "/zonemaster" "/var/www/html/zonemaster-web-gui/dist"
+```
+
```

Comment:
Here we need step three:
```
3. Update "apiEndpoint" in "assets/app.config.json" (possibly creating it). See Configuration.md and the example app.config.json
```

I guess #267 so that the text here could correctly refer to the example file.